testhelper/testhelper.go

```diff
@@ -255,9 +351,8 @@ func CreateSLAMService(
 		}
 		cameraName = cfg.Sensors[0]
 	}
-	test.That(t, sensorDeps, test.ShouldResemble, []string{cameraName})
-
-	deps := s.SetupDeps(cameraName)
+	test.That(t, lidarDeps, test.ShouldResemble, []string{cameraName})
```

Comment:
[q] where is the check of `imuDeps` if enabled?

Reply:
Again, as before, Validate is still only taking in the Lidar, not the IMU, since the IMU is being treated as an optional parameter. This was something I was originally unsure about but Esha said would be fine to do? Although it is possible that she did not have a full grasp of the task at hand when making that comment.

Reply:
Added IMU to validate to create implicit dependencies